Source/Core/VideoBackends/D3D12/D3D12BoundingBox.cpp

```diff
@@ -101,8 +101,8 @@ bool D3D12BoundingBox::CreateBuffers()
                                      D3D12_RESOURCE_FLAG_ALLOW_UNORDERED_ACCESS};
 
   HRESULT hr = g_dx_context->GetDevice()->CreateCommittedResource(
-      &gpu_heap_properties, D3D12_HEAP_FLAG_NONE, &buffer_desc,
-      D3D12_RESOURCE_STATE_UNORDERED_ACCESS, nullptr, IID_PPV_ARGS(&m_gpu_buffer));
+      &gpu_heap_properties, D3D12_HEAP_FLAG_NONE, &buffer_desc, D3D12_RESOURCE_STATE_COMMON,
```

Comment:
Do you recall what the debug layer error was?  It'd be nice if that could go into the commit

Reply:
Would be nice to do for the other one as well.

Reply:
Resources in the READBACK heap must use this specific initial state.

Reply:
I meant add the debug layer error to the commit message (for the "don't query the GPU descriptor" commit)